core/chains/evm/txmgr/broadcaster_test.go

```diff
@@ -1486,7 +1487,7 @@ func TestEthBroadcaster_ProcessUnstartedEthTxs_Errors(t *testing.T) {
 		// Do the thing
 		retryable, err := eb2.ProcessUnstartedTxs(testutils.Context(t), fromAddress)
 		require.Error(t, err)
-		require.Contains(t, err.Error(), "bumped gas price of 20 gwei is equal to original gas price of 20 gwei. ACTION REQUIRED: This is a configuration error, you must increase either EVM.GasEstimator.BumpPercent or EVM.GasEstimator.BumpMin")
+		require.Contains(t, err.Error(), "bumped fee price of 20000000000 fee unit is equal to original fee price of 20000000000 fee unit. ACTION REQUIRED: This is a configuration error, you must increase either FeeEstimator.BumpPercent or FeeEstimator.BumpMin")
```

Comment:
is there a way to still return `wei` or `gwei` instead of `fee unit` in the errors? like a chain specific way for stringifying the value?

just thinking it's much more readable with proper units and can also inform the user which chain is being used

Reply:
maybe something like passing a `FeeUnitToChainUnit` func for printing?

Reply:
good point, thanks Aaron for this. I think that having a chain specific way to stringify the values would be beneficial to the users. 

```
FeeUnitToChainUnit
		- Pass in chain type as a enum 
		- Based on the enum, it will update the units for printing based on big.Int
```